elastic-utils-lib/src/cmr/elastic_utils/search/es_index_name_cache.clj

```diff
@@ -64,6 +64,9 @@
     (hcache/set-value cache index-names-cache-key :rebalancing-collections (:rebalancing-collections index-names-map))
     (info "Refreshed search index-names cache.")))
 
+(declare ctx system)
+;TODO: figure out why RefreshIndexNamesCacheJob can not be "declare"
```

Comment:
Are you going to return to this?

Reply:
remove the TODO.  The defjob is not a variable I think that is why you can't declare it.  I am not sure what it is though - I am anticipating your next question. :-)

Reply:
I think the new config file will take care of this, otherwise, it will just be fixed next time we do a lint